frontend/src/static/css/every-menu-style.scss

```diff
@@ -59,33 +44,46 @@ export const MenuItem = styled.li`
       transform: scale(1.2);
     }
   }
-`
+}
 
-// Base container style for the menu
-export const ContainerMenu = styled.div`
+// Menu style directed to the top
+.container-top-menu {
   display: flex;
-  flex-direction: row;
-  height: 100%;
+  flex-direction: column;
   width: 100%;
-`
+}
 
-// Side-directed menu style
-export const ContainerSideMenu = styled.div`
+.container {
   display: flex;
-  flex-direction: column;
-  height: 100%;
-  width: 75px;
-  background: #2196D9;
+  align-items: center;
+  width: 100%;
+  background-color: #fff;
+  box-shadow: 0 2px 5px -2px rgba(0, 0, 0, 0.3), 0 -2px 5px -2px rgba(0, 0, 0, 0.3);
+  height: 50px;
+
+  span {
+    margin-left: 2%;
+  }
 
   ul {
     display: flex;
-    height: 100%;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
+    width: 90%;
+    justify-content: flex-end;
+    list-style-type: none;
+    margin-right: 5px;
   }
 
-  li a, li span {
-    font-size: 25px;
+  li {
+    margin-right: 25px;
+    cursor: pointer;
   }
-`
\ No newline at end of file
+}
+
+// Imagem do usuario
+.img-user {
```

Comment:
You can add it inside of selector of top menu style. It's most easy to indentify were is the image style. The sass use the hierarchy tree.